themes/classic/templates/checkout/order-confirmation.tpl

```diff
@@ -93,11 +93,31 @@
   {/block}
 
   {block name='customer_registration_form'}
-    {if $customer.is_guest}
+    {if $order_customer.is_guest && !$registered_customer_exists}
       <div id="registration-form" class="card">
         <div class="card-block">
-          <h4 class="h4">{l s='Save time on your next order, sign up now' d='Shop.Theme.Checkout'}</h4>
-          {render file='customer/_partials/customer-form.tpl' ui=$register_form}
+          <h4 class="h4">{l s='Save time on your next order, sign up now and enjoy:' d='Shop.Theme.Checkout'}</h4>
+          <ul>
+            <li> -{l s='Personalized and secure access' d='Shop.Theme.Customeraccount'}</li>
+            <li> -{l s='Fast and easy checkout' d='Shop.Theme.Customeraccount'}</li>
+            <li> -{l s='Easier merchandise return' d='Shop.Theme.Customeraccount'}</li>
+          </ul>
+          <form method="post">
```

Comment:
I'm not in favor of hardcoding theme forms, it's really good for developer to handle form markup in one place instead of looking for forms all over the place. Consider building this form in the same way like we build login form etc.

Reply:
@kpodemski It's nearly the same hardcoded form as on guest tracking. I think we could merge it into one TPL and reuse it, at least.